wpimath/src/test/java/edu/wpi/first/math/controller/ElevatorFeedforwardTest.java

```diff
@@ -22,6 +26,18 @@ void testCalculate() {
     assertEquals(4.5, m_elevatorFF.calculate(2), 0.002);
     assertEquals(6.5, m_elevatorFF.calculate(2, 1), 0.002);
     assertEquals(-0.5, m_elevatorFF.calculate(-2, 1), 0.002);
+    var r = VecBuilder.fill(2.0);
```

Comment:
Add a newline before this because it's logically separate from the preceding code.